src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -237,6 +238,8 @@ def _structure_results_from_area_object(target_area: "AreaBase") -> Dict:
         area_dict = {}
         area_dict["name"] = target_area.name
         area_dict["uuid"] = target_area.uuid
+        if target_area.is_home_area and gsy_e.constants.RUN_IN_NON_P2P_MODE:
+            area_dict["non_p2p"] = True
```

Comment:
This is the only think that raises my eyebrow in this list of PRs actually. Not a problem since it is "only" in the results, however Not certain, however it might be better if we added a global constant as a part of the results. The disadvantage of this is that the detection of "home areas" will need to happen in gsy-framework, thus being slightly more complex than here. Thus, please ignore this suggestion and if we find the need for additional logic in the results service in the future, we can rethink about this approach.

Reply:
Yeah, I was not 100% happy with this implementation either. But you suggestion also does not convince me actually. I will take your offer to ignore it. Let's resume this discussion once we stumble over issues with this, ok?